Makefile

```diff
@@ -268,7 +268,7 @@ run-android: export TARGET := android
 run-android: ##@run Build Android APK and start it on the device
 	npx react-native run-android --appIdSuffix debug
 
-SIMULATOR=iPhone 13
+SIMULATOR=iPhone 14
```

Comment:
it should be `iPhone 14 Pro` because of the screen size, its identical to figma designs

Reply:
Yeah good point, I wondered why we didn't have 13 Pro as default before, I think 14 Pro is the best option now

Reply:
i meant `iPhone 11 Pro` sorry